ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -36,18 +38,18 @@ function TokenList({ onTokenClick }: TokenListProps) {
     chainIds: chainIdsToPoll as Hex[],
   });
 
```

Comment:
In the selectors here i think we should use 
` const currentNetwork = useSelector(getMultichainNetwork);`
instead of 
` const currentNetwork = useSelector(getCurrentNetwork);`

Otherwise the chainId would have the value for the EVM chain